File: 240310/문자열 길이의 합/sum-length-of-string.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
int n;
cin>>n;
string str[10];
int cnt=0;
int len_all=0;

for(int i=0;i<10;i++){
    cin>>str[i];
}
for(int i=0;i<10;i++){
    len_all+=str[i].length();
    if(str[i][0]=='a') cnt++;
    
}


cout<<len_all<<" "<<cnt;
return 0;

}

File: 240615/문자열 길이의 합/sum-length-of-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);

    int n=sc.nextInt();
    String [] str=new String[10];

    for(int i=0;i<n;i++)
    str[i]=sc.next();

    int lenall=0;
    int cnt=0;

    for(int i=0;i<n;i++){
        int len=str[i].length();
        lenall+=len;
        if(str[i].charAt(0)=='a')
        cnt++;
    }
    System.out.print(lenall+" "+cnt);

    }
}